src/main/java/frc/robot/RobotContainer.java

```diff
@@ -21,13 +24,34 @@
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
   private DriveSystem driveSystem;
-  
+
+  private InstantCommand toggleFieldOriented; 
+  private InstantCommand toggleSlowMode;
+
   private DriveWithJoystick driveWithJoystick;
 
+  private Joystick joy;
+  private JoystickButton toggleFieldOrientedBtn;
+  private JoystickButton toggleSlowModeBtn;
   /** The container for the robot. Contains subsystems, OI devices, and commands. */
   public RobotContainer() {
+
+    joy = new Joystick(0);
+    toggleFieldOrientedBtn = new JoystickButton(joy, 5);
+    toggleSlowModeBtn = new JoystickButton(joy, 7);
+
+    driveSystem = new DriveSystem();
+
+    toggleFieldOriented = new InstantCommand(driveSystem::toggleFieldOriented, driveSystem);
+    toggleSlowMode = new InstantCommand(driveSystem::toggleSlowMode, driveSystem);
+
+    driveWithJoystick = new DriveWithJoystick(driveSystem, joy);
+    driveSystem.setDefaultCommand(driveWithJoystick);
+
```

Comment:
Letś organize this with comments.  //Subsystems //Joysticks //Buttons, //Comands